app/controllers/jobs_controller.rb

```diff
@@ -1,15 +1,22 @@
 class JobsController < ApplicationController
   def index
-    @company = Company.find(company_id)
-    @jobs = @company.jobs
+    if include_sort?
+      @jobs = Job.order(:level_of_interest).reverse
```

Comment:
this too (`order(:level_of_interest).reverse`) should be wrapped in a method on your `Job` class rather than in the controller!